helm/alfresco-process-application/templates/volume-init-job.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.persistence.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: volume-init
+spec:
+  template:
+    spec:
+      restartPolicy: "OnFailure"
+      {{- if index .Values.global "registryPullSecrets" }}
```

Comment:
where does it get the cluster registry secret from? 

Here i still see quay.io
https://github.com/Alfresco/alfresco-process-application-deployment/pull/7/files#diff-ab6750f6fe536fc16e61e43a4f929c25R4

Reply:
so the idea here is that when you deploy with pvc you can pass multiple secrets other than quay secret, so in values-with-pvc.yaml you can pass all those secret and it will override default value.

Reply:
perfect please let's document it